src/pages/components/Nav/Nav.js

```diff
@@ -1,14 +1,78 @@
 import React, { Component } from "react";
+import MenuDropBox from "./MenuDropBox/MenuDropBox";
 import "./Nav.scss";
 
 class Nav extends Component {
+  constructor() {
+    super();
+    this.state = {
+      menus: [],
+      hoveredIdx: ""
+    };
+  }
+
+  componentDidMount() {
+    fetch("http://localhost:3000/data/nav.json", { method: "GET" })
+      .then(nav => nav.json())
+      .then(nav => {
+        this.setState({
+          menus: nav.menus
+        });
+      });
+  }
+
+  //MouseOver style 변화
+  handleMenuStyle = idx => {
+    this.setState({ menus: this.state.menus, hoveredIdx: idx });
+  };
+
   render() {
+    const { menus, hoveredIdx } = this.state;
     return (
       <div>
-        
+        <div className="Nav">
+          <div className="menuTab">
+            <div>
+              <i className="fas fa-bars"></i>
+              <span>
+                monawe<span>mall</span>
+              </span>
+            </div>
+            <ul>
+              {menus.map((menu, idx) => (
+                <div
+                  key={idx}
+                  className={hoveredIdx === idx ? "activeMenu" : "inactiveMenu"}
```

Comment:
```suggestion
                  className={hoveredIdx === idx ? "menu active" : "menu inactive"}
```
클래스 셀렉터는 두 가지 이상의 중복 사용 가능하기 때문에 공통속성을 재활용하면서 달라지는 부분만 별도의 속성으로 관리할 수 있습니다

Reply:
class 이름 구분해서 style 주었습니다~